### ISSUE TITLE
Fails to compile on nightly-armv7-unknown-linux-gnueabihf in release mode

### ISSUE THREAD

**Issue by fenhl** [rust-lang/libc#1441]
Steps to reproduce:

```sh
cargo new --bin compile-fail
cd compile-fail
rustup override set nightly
echo 'libc = "=0.2.60"' >> Cargo.toml
cargo check --release
```

Output:

```
     Created binary (application) `compile-fail` package
info: using existing install for 'nightly-armv7-unknown-linux-gnueabihf'
info: override toolchain for '/home/fenhl/compile-fail' set to 'nightly-armv7-unknown-linux-gnueabihf'

  nightly-armv7-unknown-linux-gnueabihf unchanged - rustc 1.38.0-nightly (273f42b59 2019-07-21)

    Updating crates.io index
   Compiling libc v0.2.60
error: Could not compile `libc`.

Caused by:
  process didn't exit successfully: `rustc --crate-name build_script_build /home/fenhl/.cargo/registry/src/github.com-1ecc6299db9ec823/libc-0.2.60/build.rs --color always --crate-type bin --emit=dep-info,link -C opt-level=3 --cfg 'feature="default"' --cfg 'feature="std"' -C metadata=aaad41c440293f27 -C extra-filename=-aaad41c440293f27 --out-dir /home/fenhl/compile-fail/target/release/build/libc-aaad41c440293f27 -L dependency=/home/fenhl/compile-fail/target/release/deps --cap-lints allow` (signal: 11, SIGSEGV: invalid memory reference)
```

Sorry if this is a duplicate of #1427, I couldn't tell whether this toolchain is tested by the CI.

**Comment by gnzlbg** [comment]
This toolchain is tested by CI, and that appears to be a segmentation fault in the Rust compiler. Could you open an issue in rust-lang/rust and link it with this one ?

**Comment by JohnTitor** [comment]
It seems this is fixed by rust-lang/rust#62896. Feel free to let me know if the issue is still present.
